configure.ac

```diff
@@ -83,6 +83,8 @@ AC_FUNC_CLOSEDIR_VOID
 AC_FUNC_STAT
 AC_CHECK_FUNCS([memmove strncasecmp strstr strdup])
 
+AC_SEARCH_LIBS([dlopen], [dl dld])
```

Comment:
Probably should also check `dlsym`, `dlclose` and `dlerror`.

Reply:
It's mainly not about checking if `dlopen()` exists, but if we need to link with `-ldl` or `-ldld` to make the function available.